IFC4x3/Sections/Core data schemas/Schemas/IfcKernel/Entities/IfcRelAssociatesDataset/Documentation.md

```diff
@@ -0,0 +1,5 @@
+The objectified relationship (_IfcRelAssociatesDataset_) handles the assignment of a dataset information (items of the select _IfcDatasetSelect_) to objects occurrences (subtypes of _IfcObject_) or object types (subtypes of _IfcTypeObject_).
```

Comment:
The objectified relationship (_IfcRelAssociatesDataset_) handles the assignment of a dataset (items of the select _IfcDatasetSelect_) to objects occurrences (subtypes of _IfcObject_) or object types (subtypes of _IfcTypeObject_).

Reply:
The description is written in the same fashion as the parallel IfcRelAssociatesDocumentation. Both IfcDatasetInformation and IfcDatasetReference may be associated. Maybe if I write "assignment of dataset information or dataset reference (items of the select...)"?

Reply:
Committing the suggestion of @SergejMuhic below.